### PR TITLE
[NOJIRA] - playJsonOps should not depend on test

### PR DESCRIPTION
**Pull request by vodopis**
@chrissalij-r

### CODE REVIEW

**Comment by chrissalij-r** [comment]
@jeffmay @slustbader

**Comment by chrissalij-r** [comment]
tests failed to compile

**Comment by egprentice** [comment]
LGTM when test compile and run

**Comment by scalp42** [comment]
LGTM cc @arnaud036

**Comment by chrissalij-r** [comment]
HINT: Bonus points if you want to move everything down into the root. But thats optional

LGTM. @jeffmay hows it look to you?

**Comment by jeffmay** [comment]
FAIL: This will break downstream dependencies.

File: build.sbt [hunk 1]
```diff
@@ -45,22 +45,10 @@ lazy val playJsonOps = project in file("playJsonOps") settings(common: _*) setti
...
-  name := "play-json-tests",
-
-  libraryDependencies ++= Seq(
     "com.typesafe.play" %% "play-json" % "2.3.7",
-    "org.scalacheck" %% "scalacheck" % "1.12.2",
-    "org.scalatest" %% "scalatest" % "2.2.4",
-    "com.rallyhealth" %% "scalacheck-ops" % "1.1.0"
+    "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
+    "org.scalatest" %% "scalatest" % "2.2.4" % "test",
+    "com.rallyhealth" %% "scalacheck-ops" % "1.1.0" % "test"
```
Comment (jeffmay) [inline build.sbt]: What's the purpose of making this helpful testing library only available as a test.jar?

I don't think this is how it works for any other helpful testing libraries:
- https://github.com/scalatest/scalatestplus-play/blob/master/build.sbt#L30-L32

My assumption was that you would always depend on *this* library using the "test" scope.

**Comment by jeffmay** [comment]
```
23:43:08 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:6: object ops is not a member of package scala.concurrent.duration
23:43:08 [error] import scala.concurrent.duration.ops._
23:43:08 [error]                                  ^
23:43:08 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:38: not found: value DurationOps
23:43:08 [error]         DurationOps.parseLossless(str) match {
23:43:08 [error]         ^
23:43:09 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:51: not found: value DurationOps
23:43:09 [error]       DurationOps.parseLossless(str) match {
23:43:09 [error]       ^
23:43:09 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:53: type mismatch;
23:43:09 [error]  found   : Any
23:43:09 [error]  required: scala.concurrent.duration.Duration
23:43:09 [error] Error occurred in an application involving default arguments.
23:43:09 [error]           JsSuccess(duration)
23:43:09 [error]                     ^
23:43:09 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:76: not found: value DurationOps
23:43:09 [error]           DurationOps.parseLossless(str) match {
23:43:09 [error]           ^
23:43:09 [error] /ssd/jenkins/workspace/Build_play-json-ops/playJsonOps/src/main/scala/play/api/libs/json/ops/DurationFormat.scala:77: type mismatch;
23:43:09 [error]  found   : Any
23:43:09 [error]  required: scala.concurrent.duration.Duration
23:43:09 [error] Error occurred in an application involving default arguments.
23:43:09 [error]             case Success(duration) => JsSuccess(duration)
```

**Comment by jeffmay** [comment]
Also, can you put this PR up against the community version:
https://github.com/jeffmay/play-json-ops

I like all changes to flow from community to Rally.

**Comment by chrissalij-r** [comment]
Update LGTM

**Pull request closed without merging**